opencti-platform/opencti-front/src/private/components/data/ToolBar.jsx

```diff
@@ -1863,6 +1863,7 @@ class ToolBar extends Component {
                         <ItemIcon type={element.entity_type} />
                       </ListItemIcon>
                       <ListItemText
+                        style={{ overflowX: 'hidden', textOverflow: 'ellipsis' }}
```

Comment:
Not working because the style here is targeting the root element of `ListItemText` but the text is inside a `span` itself inside the root element. We need to target this `span` instead.
```suggestion
                        sx={{
                          '.MuiListItemText-primary': {
                            overflowX: 'hidden',
                            textOverflow: 'ellipsis',
                          },
                        }}
```

Reply:
`sx` is a prop of MaterialUI and we can pass objects with css classes as keys to target elements inside the component